packages/web/src/pages/history-page/components/desktop/HistoryPage.tsx

```diff
@@ -61,14 +60,12 @@ const HistoryPage = ({
 
   const playAllButton = !loading ? (
     <Button
-      className={styles.playAllButton}
-      textClassName={styles.playAllButtonText}
-      iconClassName={styles.playAllButtonIcon}
-      type={ButtonType.PRIMARY_ALT}
-      text={queuedAndPlaying ? 'PAUSE' : 'PLAY'}
-      leftIcon={queuedAndPlaying ? <IconPause /> : <IconPlay />}
+      css={(theme) => ({ marginLeft: theme.spacing.xl })}
+      iconLeft={queuedAndPlaying ? IconPause : IconPlay}
       onClick={onPlay}
-    />
+    >
+      {queuedAndPlaying ? 'play' : 'pause'}
```

Comment:
Not sure if these should be capitalized
```suggestion
      {queuedAndPlaying ? 'Play' : 'Pause'}
```